megago/megago.py

```diff
@@ -30,7 +32,7 @@
 
 GODAG_FILE_PATH = os.path.join(DATA_DIR, "go-basic.obo")
 UNIPROT_ASSOCIATIONS_FILE_PATH = os.path.join(DATA_DIR, "associations-uniprot-sp-20200116.tab")
-
+JSON_INDEXED_FILE_PATH = os.path.join(DATA_DIR, "go_freq_uniprot.json")
```

Comment:
move to `__init__.py` and load from there. Probably makes sense to define the data dir there, too.